packages/grafana-ui/src/internal/index.ts

```diff
@@ -63,9 +63,10 @@ export {
   FILTER_FOR_OPERATOR,
   FILTER_OUT_OPERATOR,
 } from '../components/Table/types';
-export { defaultSparklineCellConfig } from '../components/Table/SparklineCell';
-export { TableCell } from '../components/Table/TableCell';
-export { useTableStyles } from '../components/Table/styles';
+export { defaultSparklineCellConfig } from '../components/Table/Cells/SparklineCell';
+export { TableCell } from '../components/Table/Cells/TableCell';
+export { useTableStyles } from '../components/Table/TableRT/styles';
+export { Trans } from '../utils/i18n';
```

Comment:
This shouldn't be exported from @grafana/ui use the `<Trans />` component in https://github.com/grafana/grafana/blob/main/public/app/core/internationalization/index.tsx#L78 instead. Thanks 🙏